userspace/libsinsp/container.cpp

```diff
@@ -271,7 +271,8 @@ std::string sinsp_container_manager::container_to_json(const sinsp_container_inf
 
 bool sinsp_container_manager::container_to_sinsp_event(const std::string& json, sinsp_evt* evt, std::shared_ptr<sinsp_threadinfo> tinfo, char *scap_err)
 {
-	size_t totlen = sizeof(scap_evt) + sizeof(uint32_t) + json.length() + 1;
+	uint32_t json_len = json.length() + 1;
+	size_t totlen = sizeof(scap_evt) + sizeof(uint32_t) + json_len;
```

Comment:
is there are reason behind this change?

Reply:
Just a left over of a debugging session 😅 let me fix it.

Reply:
uhm got it!